### PR TITLE
Update is_funky flags in curated records

### PR DESCRIPTION
**Pull request by stepps00**
Fixes whosonfirst-data/whosonfirst-data#1605, replaces https://github.com/whosonfirst-data/whosonfirst-data-admin-us/pull/8.

Updates all mz:is_funky property values to 0 in records that we've previously curated through a neighbourhood update issue. Does not require PIP work.

### CODE REVIEW

**Comment by stepps00** [comment]
Note: `data/102/061/079/102061079.geojson` stays as-is.

File: data/858/346/37/85834637.geojson [hunk 1]
```diff
@@ -96,7 +96,7 @@
         }
     ],
     "wof:id":85834637,
-    "wof:lastmodified":1527270119,
+    "wof:lastmodified":1559246351,
     "wof:name":"Inner Mission",
```
Comment (nvkelso) [inline data/858/346/37/85834637.geojson]: Please let's call this `The Mission` in the label for English

**Comment by stepps00** [comment]
d9a163d adds a label to the Inner Mission record and rests all microhood min_zooms to 16.0 if the value was lower than 16 (17+ min_zooms stay as-is).

File: data/907/212/589/907212589.geojson [hunk 2]
```diff
@@ -21,7 +21,7 @@
     "mz:is_landuse_aoi":0,
     "mz:is_official":0,
     "mz:max_zoom":18.0,
-    "mz:min_zoom":12.0,
```
Comment (nvkelso) [inline data/907/212/589/907212589.geojson]: Please reset this one to 14.

File: data/907/203/481/907203481.geojson [hunk 3]
```diff
@@ -21,7 +21,7 @@
     "mz:is_landuse_aoi":0,
     "mz:is_official":0,
     "mz:max_zoom":18.0,
-    "mz:min_zoom":13.0,
```
Comment (nvkelso) [inline data/907/203/481/907203481.geojson]: Please reset this one to 14.

**Comment by nvkelso** [comment]
I don't see the Mission label changes, though?

**Comment by stepps00** [comment]
Newest comment addresses the min_zoom changes (and actually sets the Mission label).

**Pull request merged**